changelog.md

```diff
@@ -0,0 +1,7 @@
+# Changelog
+
+## Unreleased
+
+### Fixed
```

Comment:
Please remove the changelog entry from this PR (then you can go ahead and merge it directly). @dragonstyle and I are talking now about a way to incorporate changelog entries into commits without merge problems.